fedimint-server/src/db.rs

```diff
@@ -48,27 +72,33 @@ impl_db_lookup!(
     query_prefix = AcceptedTransactionKeyPrefix
 );
 
-#[derive(Debug, Copy, Clone, Encodable, Decodable, Serialize)]
-pub struct EpochHistoryKey(pub u64);
+#[derive(Debug, Encodable, Decodable)]
+pub struct SignedBlockKey(pub u64);
 
 #[derive(Debug, Encodable, Decodable)]
-pub struct EpochHistoryKeyPrefix;
+pub struct SignedBlockPrefix;
 
 impl_db_record!(
-    key = EpochHistoryKey,
-    value = SignedEpochOutcome,
-    db_prefix = DbKeyPrefix::EpochHistory,
+    key = SignedBlockKey,
+    value = SignedBlock,
+    db_prefix = DbKeyPrefix::SignedBlock,
+    notify_on_modify = true,
 );
-impl_db_lookup!(key = EpochHistoryKey, query_prefix = EpochHistoryKeyPrefix);
+impl_db_lookup!(key = SignedBlockKey, query_prefix = SignedBlockPrefix);
 
-#[derive(Debug, Encodable, Decodable, Serialize)]
-pub struct LastEpochKey;
+#[derive(Debug, Encodable, Decodable)]
+pub struct AlephUnitsKey(pub u64, pub u64);
```

Comment:
I have no idea what it is, and it is composed of two unnamed integers. Ideally it needs named fields and docs.

Reply:
Its the backup for aleph bft to recover from a crash

Reply:
Put it in the comment / field names. :)